modules/microshift-blocking-nodeport-access.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-networking.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-blocking-nodeport-access_{context}"]
+= Blocking external access to NodePort service on a specific host interface
+
+OVN-Kubernetes does not restrict the host interface, therefore NodePort services can be accessed from outside a MicroShift node. The following procedure explains how to block the NodePort service on a specific host interface and restrict external access.
+
+.Prerequisites 
+//Not sure what prereqs are needed
```

Comment:
I couldn't tell if there were prerequisites needed for this procedure. Or rather I couldn't find any